packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -28,19 +29,25 @@ class RowLarge extends React.Component {
 
 	renderKeyValue(field, fieldIndex) {
 		const { index, parent } = this.props;
-		const cellContent = renderCell(index, parent, field);
-		const tooltip = typeof cellContent === 'string' ? cellContent : null;
-		const label = getLabel(field);
-		return (
-			<div className={theme['field-group']} role="group" key={label || index}>
-				<dt key={fieldIndex} className={theme['field-label']}>
-					{label}
-				</dt>
-				<dd className={theme['field-value']} title={tooltip}>
-					{cellContent}
-				</dd>
-			</div>
-		);
+		const rawContent = getRowData(parent, index);
+		const dataKey = getDataKey(field);
+		const value = rawContent[dataKey];
+		if (value != null && `${value}`.trim().length) {
```

Comment:
Why checking if need to display based on `rawContent` when its `cellContent` which is used ?

Reply:
In my case, `cellContent` contains extra div with empty content.
I can't lead on.

Reply:
yep nevermind, I didn't chek the apis ;)